app/views/hub/clients/edit_13614c_form.html.erb

```diff
@@ -69,7 +69,7 @@
       <div id="marital-status-fields">
         <h2>Part II – Marital Status and Household Information</h2>
         <hr>
-        <h3><%= t(".what_was_your_marital_status", current_tax_year: TaxReturn.current_tax_year) %></h3>
+        <h3><%= t(".what_was_your_marital_status", current_tax_year: MultiTenantService.new(:gyr).current_tax_year) %></h3>
```

Comment:
could possibly be `gyr_service.current_tax_year` but you cannot believe how non-strongly i feel